e2e_test/e2e_test.go

```diff
@@ -350,8 +349,6 @@ var _ = g.Describe("E2E integration testing", g.Ordered, func() {
 			// This should be the IP we expect to be able to reach from the DPU's secondary network
 			externalClientIp, err = getEnv("EXTERNAL_CLIENT_IP")
 			Expect(err).NotTo(HaveOccurred())
-			// This should be the dev on the external client that is connected to the DPU's secondary network
-			externalClientDev, err = getEnv("EXTERNAL_CLIENT_DEV")
```

Comment:
it would be useful if the commit message would explain why this is done.

Is there a problem with the test? Obviously there is, but which is the problem?

Depending on that, it tells whether this test should ever be restored (in a similar form and how to changed it).

Reply:
No idea what's wrong. I don't this this should have ever worked.

We've discussed this with Intel. They will revert the removal in a PR that fixes the behavior.